zaza/charm_tests/glance/setup.py

```diff
@@ -52,11 +56,13 @@ def add_lts_image(glance_client=None):
         keystone_session = openstack_utils.get_overcloud_keystone_session()
         glance_client = openstack_utils.get_glance_session_client(
             keystone_session)
-    image_url = openstack_utils.find_ubuntu_image(
-        release='bionic',
-        arch='amd64')
-    print(image_url)
-    openstack_utils.create_image(
-        glance_client,
-        image_url,
-        'bionic')
+    if openstack_utils.get_images_by_name(glance_client, 'bionic'):
```

Comment:
It really surprises me that bionic is hard coded here. The name and/or series should be parameters. "lts" could be trusty, xenial, bionic or "F"